rollup.build.js

```diff
@@ -63,7 +63,7 @@ const configs = {
 }
 
 const outputs = {
-  bundle: [output('umd'), output('esm')],
+  bundle: [output('umd'), output('cjs'), output('esm')],
```

Comment:
Currently, all bundles are the same, just in different directories, right?

Reply:
Yep: `index.all.js` + `index.js` + minified versions of those + sourcemaps